BUILD

```diff
@@ -468,7 +474,6 @@ grpc_cc_library(
             "//conditions:default": GRPC_XDS_TARGETS,
         },
     ],
-    standalone = True,
```

Comment:
what does this change?

Reply:
Nothing.

I found the parameter is unused in both Github and google3.

https://github.com/grpc/grpc/blob/master/bazel/grpc_build_system.bzl#L145
https://source.corp.google.com/piper///depot/google3/third_party/grpc/bazel/grpc_build_system.bzl;l=657